lib/DefinePlugin.js

```diff
@@ -249,8 +255,12 @@ const toCacheVersion = code => {
 	return code + "";
 };
 
-const VALUE_DEP_PREFIX = "webpack/DefinePlugin ";
-const VALUE_DEP_MAIN = "webpack/DefinePlugin_hash";
+const PLUGIN_NAME = "DefinePlugin";
+const VALUE_DEP_PREFIX = `webpack/${PLUGIN_NAME} `;
+const VALUE_DEP_MAIN = `webpack/${PLUGIN_NAME}_hash`;
+const TYPEOF_OPERATOR_REGEXP = /^typeof\s+/;
+const WEBPACK_REQUIRE_FUNCTION_REGEXP = /__webpack_require__\s*(!?\.)/;
+const WEBPACK_REQUIRE_IDENTIFIER_REGEXP = /__webpack_require__/;
```

Comment:
We can reuse it from https://github.com/webpack/webpack/blob/main/lib/RuntimeGlobals.js#L11

Reply:
Yup, that is going to be my next goal. We can leave for now, but thats my next refactoring.

Reply:
__webpack_require__ token strings literals are _everywhere_

Reply:
Created an issue from your comment and will resolve this.